README.md

```diff
@@ -172,6 +172,16 @@ w('my-widget', properties, children);
 The example above that uses a string for the `widgetConstructor `, is taking advantage of our [widget registry](#widget-registry) functionality.
 The widget registry allows for the lazy instantiation of widgets.
 
+### Responding to Render Events
+
+Sometimes you may want to be notified when a widget is rendered. Widgets will emit a `render:rendered` event with the result of the render operation.
```

Comment:
This is a bit wishy-washy... we should either say why you might want to be notified, or just state something like "Widgets will emit a `render:rendered` event with the result of the render operation. Listen to this event to be notified."